docs/reference/tasking-reference.md

```diff
@@ -1,7 +1,227 @@
 # Tasking
 
-::: up42.tasking.Tasking
-    selection:
-        inherited_members: True
-        members: {{ funcs_tasking }}
-        
\ No newline at end of file
+The Tasking class enables access to the UP42 [tasking functionality](../tasking.md).
+
+```python
+tasking = up42.initialize_tasking()
+```
+
+## Order
+
+### construct_order_parameters
+
+The `construct_order_parameters` function allows you to fill out an order form for a new tasking order.
+
+```python
+construct_order_parameters(
+    data_product_id,
+    name,
+    acquisition_start,
+    acquisition_end,
+    geometry
+)
+```
+
+<h3> Arguments </h3>
+
+| Name                | Type                                                                          | Description                                                                                                                                   |
+| ------------------- | ----------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------- |
+| `data_product_id`   | `str`                                                                         | The data product ID.                                                                                                                          |
+| `name`              | `str`                                                                         | The tasking order name.                                                                                                                       |
+| `acquisition_start` | `Union[str, datetime]`                                                        | The start date of the acquisition period in the `YYYY_MM_DD` format.                                                                          |
```

Comment:
@dustgalactic the format is incorrect.
It should be "YYYY-MM-DD".

Please see [this](https://github.com/up42/up42-py/blob/f0005fd569de57d43746acf4dbadcffbb18d2ae3/up42/tasking.py#L57) for reference.

Same for acquisition_end.

Reply:
Thank you for catching that. I'll fix in a new PR

Reply:
https://up42.atlassian.net/browse/DOC-616